src/queries/meeting/useMeetingDetailInfoQuery/index.tsx

```diff
@@ -0,0 +1,17 @@
+import MeetingAPI from '@/apis/Meeting';
+import { useQuery } from '@tanstack/react-query';
+import { APIMeetingDetail } from '@/types/meetingDetail';
+
+const useMeetingDetailInfoQuery = ({
+  bookGroupId,
+}: {
+  bookGroupId: APIMeetingDetail['bookGroupId'];
+}) => {
+  return useQuery(['meetingDetailInfo', bookGroupId], () =>
+    MeetingAPI.getMeetingDetailInfo({ bookGroupId }).then(
+      response => response.data
```

Comment:
p3;

꺼내쓰는 것도 좋을 것 같아요!
```suggestion
      ({data}) => data
```